src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty lines as per the checklist item 'Don't begin class or method implementation with an empty line.'